utils/node_local_test.py

```diff
@@ -3297,28 +3342,43 @@ def run_in_fg(server, conf):
     Block until ctrl-c is pressed.
     """
 
-    pool = server.get_test_pool()
+    pool = server.get_test_pool_obj()
+    label = 'foreground_cont'
+    container = None
 
-    dfuse = DFuse(server, conf, pool=pool)
-    dfuse.start()
+    conts = pool.fetch_containers()
+    for cont in conts:
+        if cont.label == label:
+            container = cont.uuid
+            break
 
-    container = create_cont(conf, pool, ctype="POSIX")
+    if not container:
+        container = create_cont(conf, pool.uuid, label=label, ctype="POSIX")
+
+    dfuse = DFuse(server, conf, pool=pool.uuid, caching=True, wbcache=False)
+    dfuse.start()
 
     run_daos_cmd(conf,
                  ['container', 'set-attr',
-                  pool, container,
-                  '--attr', 'dfuse-direct-io-disable', '--value', 'on'],
+                  pool.label, container,
+                  '--attr', 'dfuse-direct-io-disable', '--value', 'off'],
+                 show_stdout=True)
+    run_daos_cmd(conf,
+                 ['container', 'set-attr',
+                  pool.label, container,
+                  '--attr', 'dfuse-data-cache', '--value', 'off'],
                  show_stdout=True)
 
     t_dir = join(dfuse.dir, container)
 
     print('Running at {}'.format(t_dir))
-    print('daos container create --type POSIX ' \
-          '{} --path {}/uns-link'.format(
-              pool, t_dir))
-    print('cd {}/uns-link'.format(t_dir))
+    print('export PATH={}:$PATH'.format(os.path.join(conf['PREFIX'], 'bin')))
+    print('export LD_PRELOAD={}'.format(os.path.join(conf['PREFIX'], 'lib64', 'libioil.so')))
+    print('export DAOS_AGENT_DRPC_DIR={}'.format(conf.agent_dir))
+    print('export D_IL_REPORT=-1')
+    print('daos container create --type POSIX --path {}/uns-link'.format(t_dir))
     print('daos container destroy --path {}/uns-link'.format(t_dir))
-    print('daos cont list {}'.format(pool))
+    print('daos cont list {}'.format(pool.label))
```

Comment:
I don't understand what this section is doing. Is this just informational about what was already ran?

Reply:
This is in the "launch" command where you run NLT and it starts dfuse and waits for it to exit, then in another window you can interact with dfuse, and NLT will do the shutdown/log analysis after you fusermount it.  It's a workflow I use a lot, and NLT currently prints out the container data and mount-point and some commands you can cut-and-paste into the other window.

The change here is a big step forward in usability for this workflow which is what I was using when developing this test.